src/app/algconfig/algconfig.component.html

```diff
@@ -0,0 +1,60 @@
+<div class="accordion accordion-flush" id="algorithmConfigurations">
```

Comment:
This component need to be dynamic based on the algorithm category and what the actual algorithm is